src/styles/main.scss

```diff
@@ -0,0 +1,5 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/normalize";
+@import "./blocks/stopwatch";
+@import "./utils/keyframes";
```

Comment:
It's recommended to import keyframes before other blocks, as they might be used within those blocks.